platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ConfigurationMigrationManager.java

```diff
@@ -318,7 +336,7 @@ private MigrationReport doImport(
       report.record(new MigrationException(Messages.IMPORT_INTERNAL_ERROR, exportFile, e));
     }
     report.end();
-    if ((zip == null) || (report.hasErrors())) {
+    if ((zip == null) || report.hasErrors()) {
```

Comment:
Why do we remove the extra parenthesis on the second clause and not the first?

Reply:
My formatter did it and it is also my preference. 
In the first case, it wraps an operation and avoids the reader having to worry about the priority order of operations. In the second case, it is a simple method call so the extra parenthesizes are not adding anything.